tests/helpers/start-app.js

```diff
@@ -1,21 +1,21 @@
 import Ember from 'ember';
-import slregisterTestHelpers from './sl/register-test-helpers';
+import slRegisterTestHelpers from './sl/register-test-helpers';
```

Comment:
I suppose we can make this change.....but it was automatically inserted by the sl-ember-test-helpers addon.  At the very least an issue should be created there to make this modification in its blueprint.